test/e2e/testdata/containerd_calico_external.yaml

```diff
@@ -15,7 +12,8 @@ cloudProvider:
   external: true
 
 addons:
-  enable: true
   addons:
-  - name: default-storage-class
-  - name: calico-vxlan
+    - addon:
+        name: default-storage-class
```

Comment:
We should enable this addon by default in v1beta3.

Reply:
yes, in a dedicated PR. There is #2708 for this case.